### PR TITLE
evince 3.32.0

### PR DESCRIPTION
**Pull request by tschoonj**
Also:

* Add new formula `appstream-glib`, which is now a build-time requirement of evince (see https://gitlab.gnome.org/GNOME/evince/issues/1106)

### CODE REVIEW

**Comment by tschoonj** [comment]
@MikeMcQuaid It would appear that my change yesterday to produce uuid pkg-config files does not work on the Mojave VM... it does work fine locally though as I installed the macOS_SDK_headers_for_macOS_10.14.pkg. 

How do we deal with this in formulas?

**Comment by MikeMcQuaid** [comment]
> How do we deal with this in formulas?

I don't know, I don't think there's anything we can really do here. See https://github.com/Homebrew/brew/issues/5068

**Comment by tschoonj** [comment]
Thanks Mike, wasn't aware of that thread... I will fix this by letting util-linux build libuuid and then adding this as dependency for Mojave and newer...

**Comment by MikeMcQuaid** [comment]
@tschoonj Honestly I think it's worth just keeping it consistent between OS versions.

**Comment by tschoonj** [comment]
Any thoughts on how to fix the following audit error for util-linux? Make it keg-only?

```
12:44:00 Error: 1 problem in 1 formula detected
12:44:00 util-linux:
12:44:00   * Header files that shadow system header files were installed to "/usr/local/opt/util-linux/include"
12:44:00     The offending files are:
12:44:00       /usr/local/opt/util-linux/include/uuid/uuid.h
```

**Comment by MikeMcQuaid** [comment]
Make one (or both) of the formulae keg-only instead of conflicting.

File: Formula/util-linux.rb [hunk 1]
```diff
@@ -11,7 +12,7 @@ class UtilLinux < Formula
     sha256 "02c15639cc3e40c7553dbcb23e94a5e679feeaad8881ed6ddce91c4cb0a015b2" => :sierra
   end
 
-  conflicts_with "rename", :because => "both install `rename` binaries"
+  keg_only :provided_by_macos, "macOS provides the uuid.h header"
```
Comment (MikeMcQuaid) [inline Formula/util-linux.rb]: Not really provided by macOS in this case as much as a single header is?

Comment (tschoonj) [reply Formula/util-linux.rb]: Well macOS provides both header and libuuid, which is compiled into libSystem.dylib. I agree that util-linux provides other things as well, that are not provided by macOS, but I don't see what else I can do to get rid of the audit error.

Comment (MikeMcQuaid) [reply Formula/util-linux.rb]: Remove `:provided_by_macos, `

Comment (tschoonj) [reply Formula/util-linux.rb]: Done

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**